packages/create-pantheon-decoupled-kit/src/templates/gatsby-wp/package.json.hbs

```diff
@@ -37,14 +39,24 @@
 	},
 	"devDependencies": {
 		"@babel/core": "^7.20.12",
+		"@pantheon-systems/decoupled-kit-configs": "0.1.0-canary.0",
+		"@pantheon-systems/eslint-config-decoupled-kit": "0.1.0-canary.0",
```

Comment:
This is fine but will need to be changed once the non-canary version is released.. We can use `"latest"` here for the version to get around this.